### DISCUSSION TITLE
Hola buen dia, soy nuevo en esto, alguien me podria indicar como usar el modo hindsight en la gopro12. muchas gracias

### DISCUSSION THREAD

**Discussion by Juanmfunes12** [gopro/labs#985] [General]
### Discussed in https://github.com/gopro/labs/discussions/557

<div type='discussions-op-text'>

<sup>Originally posted by **FlyTieGuy** September 21, 2023</sup>
GoPro Labs on the hero 12.
The code for hindsight !MHSTO=60 worked on my GoPro 11 Black
But on the 12 it stops working again after 15 minutes, when I set to 60 minutes. It’s as if the code hasn’t registered at all. On the 11 when the camera is activated there is a message which states Features enabled and the code for Hindsight appears but that isn’t happening on the 12. This is an important feature for me so I’d appreciate a resolution.</div>

**Comment by dnewman-gpsw** [comment]
Is this a Labs question about hacks to Hindsight or just general usage of the Hindsight feature?
